planet/src/main/webapp/js/placesItinerary.js

```diff
@@ -254,16 +254,6 @@ async function submitPlace(placeId) {
         order: order,
     });
 
```

Comment:
We will need to call addVisitor here if the user is making edits on a saved list (i.e not currentList)

Reply:
Done!